examples/trusted_information_retrieval/config/config.toml

```diff
@@ -1 +1,9 @@
-database_url = "https://localhost:8888"
+name = "trusted_information_retrieval"
```

Comment:
What is the name for?

Reply:
To assign here:
https://github.com/project-oak/oak/blob/c15f2858377f6944f0c0fce4d31b2481e70d8518/oak/proto/application.proto#L54-L58